amy/autoemails/fixtures/templates_triggers.json

```diff
@@ -190,7 +190,7 @@
             "from_header": "amy-no-reply@carpentries.org",
             "cc_header": "",
             "bcc_header": "",
-            "reply_to_header": "{{ regional_coordinator_email.0 }}",
+            "reply_to_header": "",
```

Comment:
Note to myself: this change will require updating template manually in production.